### ISSUE TITLE
Time for a release?

### ISSUE THREAD

**Issue by utkarsh2102** [swig/swig#2029]
Hi @ojwb,

Thanks a bunch for working on adding support for PHP 8! 🌮 

Now that it has been already added and merged, can we see a new release for swig so that other projects (cf: https://github.com/owfs/owfs/issues/89 & https://github.com/MapServer/MapServer/issues/6201) can also build against PHP 8?

Thanks in advance! \o/

**Comment by ojwb** [comment]
If you check the milestones you'll see there's already a release planned for June.

If you want to help that happen on time, please test with SWIG from git and report any problems.

**Comment by Arfrever** [comment]
[swig-4.1 milestone](https://github.com/swig/swig/milestone/3) is still set to 2021-06-30 :) .
Any updates on planned date of release?

**Comment by ojwb** [comment]
SWIG is a volunteer project so take such milestone dates with a pinch of salt.  I've bumped the milestone date (to a pretty much arbitrary date, so don't read anything particular into the new date either - clearly we aren't going to release last June though).  Also it's William who makes the releases, not me.

There is progress towards 4.1.0 - we've addressed a number of the open issues that we're aiming to address for it, but we aren't there yet.  

As I suggested above, if you want to make a release happen then testing and reporting issues is very helpful.  If you're able to provide clean patches with tests and documentation to address those issues, even better.
